pylua/client3.py

```diff
@@ -0,0 +1,12 @@
+import asyncio
+import websockets
+from struct import *
+
+async def hello():
+	async with websockets.connect('ws://localhost:8087') as websocket:
+		message = pack('hhl',1,2,3)
+		await websocket.send(message)
+		response = await websocket.recv()
+		websocket.close()
```

Comment:
@oherasym,  are you sure that close is required? You're using "with" statement. It was invented to call close automatically